### PR TITLE
Errorpage 2

### PR DESCRIPTION
**Pull request by yeshasimaria**
@jkcheong92 I tried re-doing it. I hope this is better. The file changes here are correct, but the commit summary includes commits not made by me!

### CODE REVIEW

**Comment by jkcheong92** [comment]
@yesha. You didnt pull in from develop branch. This branch with derek's commit is outdated. You must pull in from develop since I only just updated it. Then make a new branch from there

**Comment by yeshasimaria** [comment]
@jkcheong92 haha I just pulled from develop a few mins ago before I created this branch. It reflected at the branch is up to date with origin/develop. But dunno why I still see those 3 commits! :( Do you want me to redo again? :O

**Comment by jkcheong92** [comment]
@yeshasimaria Hmm, that's weird. Can try again? This time to be sure, when you pull from develop  use the command <git pull origin develop>.

**Comment by yeshasimaria** [comment]
@jkcheong92 yep, I did use the command. But let me try again and see if it works.

**Comment by yeshasimaria** [comment]
@jkcheong92 I pulled from develop, created branch and pushed it with my work, but I still see derek's commits there. :O :O

**Comment by jkcheong92** [comment]
@yeshasimaria Okay, i work this out with you tomorrow when we meet. Wnt be merging in your changes for today then. Gna close this branch first.

**Comment by yeshasimaria** [comment]
Okay sure. Thanks
